### PR TITLE
Implement dsIS to iTunes Connect team selection

### PR DESCRIPTION
**Pull request by KrauseFx**
Fixes https://github.com/fastlane/fastlane/issues/6711
Fixes https://github.com/fastlane/fastlane/issues/6710

Thanks @summera for providing a solution for the problem

### CODE REVIEW

**Comment by webpolis** [comment]
@KrauseFx you rock!

**Review by milch — APPROVED** [review]
👍

File: spaceship/lib/spaceship/tunes/tunes_client.rb [hunk 1]
```diff
@@ -87,7 +87,10 @@ def team_id=(team_id)
 
       response = request(:post) do |req|
         req.url "ra/v1/session/webSession"
-        req.body = { contentProviderId: team_id }.to_json
+        req.body = {
+          contentProviderId: team_id,
+          dsId: Time.now.to_i.to_s # https://github.com/fastlane/fastlane/issues/6711
```
Comment (summera) [inline spaceship/lib/spaceship/tunes/tunes_client.rb]: @KrauseFx don't you want to call `user_detail_data.ds_id` here

Comment (KrauseFx) [inline spaceship/lib/spaceship/tunes/tunes_client.rb]: @summera it works with a random number too, right? By fetching the actual ds_id the request is a slower, as it has to fetch all the Team IDs.

Comment (KrauseFx) [inline spaceship/lib/spaceship/tunes/tunes_client.rb]: What do you think?

**Comment by KrauseFx** [comment]
Moved to https://github.com/fastlane/fastlane/pull/6715

File: spaceship/lib/spaceship/tunes/tunes_client.rb (hunk 1 above)
Comment (milch) [inline spaceship/lib/spaceship/tunes/tunes_client.rb]: Let's keep an eye on this, the fact that it works with a random number might point to a bug or a workaround at 's end.

Comment (summera) [inline spaceship/lib/spaceship/tunes/tunes_client.rb]: @milch my thoughts exactly. may work now, but not in future. i think it's worth doing right if it doesn't slow things down too much, which I don't think it does. @KrauseFx merged in #6715 which uses the dsId from user details

Comment (KrauseFx) [inline spaceship/lib/spaceship/tunes/tunes_client.rb]: Great think, I merged your PR, much cleaner. Thanks for your contribution. Are you on Twitter?

Comment (summera) [inline spaceship/lib/spaceship/tunes/tunes_client.rb]: Of course! Glad to help. Thanks for the quick response and fast merge to fix. 

I am: https://twitter.com/summ3ra

**Pull request closed without merging**